### ISSUE TITLE
7.4节偏导公式有误

### ISSUE THREAD

**Issue by joinlog** [dragen1860/Deep-Learning-with-TensorFlow-book#103]
测试版（20191108）
7.4 损失函数梯度
图 7.6 Softmax 函数 的下一页
“拆分为 2 部分”的公式，=后面第一个分子ezj应为ezi

**Comment by dragen1860** [comment]
收到，谢谢！
